### DISCUSSION TITLE
Equation in FAQ Part 5, ratio is inverted

### DISCUSSION THREAD

**Discussion by sonic-icons** [Anarios/return-youtube-dislike#737] [General]
Hi, it looks to me like the ratio in the FAQ Part 5 equation, describing calculation of RYD Dislike Count, is inverted. The FAQ shows (RYD Users Like Count / RYD Users Dislike Count) but it should be the inverse, (RYD Users Dislike Count / RYD Users Like Count)

**Comment by sy-b** [comment]
duplicate of https://github.com/Anarios/return-youtube-dislike/issues/725

https://github.com/Anarios/return-youtube-dislike/pull/726 fixes it

**Reply by sonic-icons** [reply]
Thank you for pointing that out.
